qiskit/aqua/components/neural_networks/quantum_generator.py

```diff
@@ -308,6 +310,45 @@ def objective_function(params):
 
         return objective_function
 
+    def _get_analytical_loss_gradient(self, quantum_instance, discriminator):
+        """
+        Get analytical loss gradient
+
+        Args:
+            quantum_instance (QuantumInstance): used to run the quantum circuit.
+            discriminator (torch.nn.Module): discriminator network to compute the sample labels.
+
+        Returns:
+            analytical_loss_gradient: function that computes analytical loss gradients.
+        """
+
+        def analytical_loss_gradient(initial_point):
+            """
+            Analytical loss gradient
+
+            Args:
+                initial_point (np.ndarray): Initial values for the variational parameters.
```

Comment:
Is this really initial point or just the current point of the variational parameters?

Reply:
It should be the current point of the variational parameters.